testDesign/tamaCode.js

```diff
@@ -1288,7 +1288,7 @@ function childOneMovement() {
 }
 
 function updateChracterPicture() {
-  if (state.tamaDead != true) {
+  if ((state.tamaDead = true)) {
```

Comment:
this is a bug. the single = is going to assign the tamaDead value to true everything this is hit, which is every time